server/src/test/java/org/elasticsearch/common/time/JavaDateMathParserTests.java

```diff
@@ -265,8 +262,9 @@ public void testIllegalMathFormat() {
     }
 
     public void testIllegalDateFormat() {
-        assertParseException("Expected bad timestamp exception", Long.toString(Long.MAX_VALUE) + "0", "failed to parse date field");
-        assertParseException("Expected bad date format exception", "123bogus", "Unrecognized chars at the end of [123bogus]");
+        // TODO FIXME
```

Comment:
Does it make sense to address this now?

Reply:
aaah. fixed already, just uncommented again